### ISSUE TITLE
Load Lotus Model node is missing

### ISSUE THREAD

**Issue by KillyTheNetTerminal** [kijai/ComfyUI-Lotus#1]
![image](https://github.com/user-attachments/assets/a4216728-b8ad-40a9-b578-dbe91cb14612)

**Comment by kijai** [comment]
No idea why that would be, does the example workflow work?

**Comment by guahunyo** [comment]
> No idea why that would be, does the example workflow work?不知道为什么会这样，示例工作流程有效吗？

Unable to import workflow
![image](https://github.com/user-attachments/assets/f103affd-f453-4795-81c3-630660584032)

**Comment by kijai** [comment]
> > No idea why that would be, does the example workflow work?不知道为什么会这样，示例工作流程有效吗？
> 
> Unable to import workflow ![image](https://private-user-images.githubusercontent.com/18573180/375314187-f103affd-f453-4795-81c3-630660584032.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg1NTQxNzgsIm5iZiI6MTcyODU1Mzg3OCwicGF0aCI6Ii8xODU3MzE4MC8zNzUzMTQxODctZjEwM2FmZmQtZjQ1My00Nzk1LTgxYzMtNjMwNjYwNTg0MDMyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDEwVDA5NTExOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJhNzcyM2UzYWQzZjcxN2I2MmU1YTIzN2JkYjlmYzBlZThkNGQzYjgzZjJjZjUzMWM5MWMyZGMwZjgyMTc2MjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.LCNeDHSYOVvC8FVNmPUnB7C0H1R6wtEZfQ6jIV3anOA)

Right, I actually forgot to upload the example workflows, they are available now.

**Comment by guahunyo** [comment]
> > > No idea why that would be, does the example workflow work?不知道为什么会这样，示例工作流程有效吗？
> > 
> > 
> > Unable to import workflow ![image](https://private-user-images.githubusercontent.com/18573180/375314187-f103affd-f453-4795-81c3-630660584032.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg1NTQxNzgsIm5iZiI6MTcyODU1Mzg3OCwicGF0aCI6Ii8xODU3MzE4MC8zNzUzMTQxODctZjEwM2FmZmQtZjQ1My00Nzk1LTgxYzMtNjMwNjYwNTg0MDMyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDEwVDA5NTExOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJhNzcyM2UzYWQzZjcxN2I2MmU1YTIzN2JkYjlmYzBlZThkNGQzYjgzZjJjZjUzMWM5MWMyZGMwZjgyMTc2MjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.LCNeDHSYOVvC8FVNmPUnB7C0H1R6wtEZfQ6jIV3anOA)
> 
> Right, I actually forgot to upload the example workflows, they are available now.

![image](https://github.com/user-attachments/assets/36ec89ef-e12b-4015-a803-03307ed8a62e)
ok now

**Comment by KillyTheNetTerminal** [comment]
![image](https://github.com/user-attachments/assets/8dff92e6-b8eb-4dea-80d6-94e7d118b5e5)
Still missing for me

**Comment by kijai** [comment]
Probably need to update ComfyUI itself.
